src/elm/Pages/Deployments/View.elm

```diff
@@ -258,3 +274,37 @@ redeployLink org repo deployment =
         ]
         [ text "Redeploy"
         ]
+
+
+{-| pullBuildLinks : takes deployment and creates a list of links to every build in the builds field
+-}
+pullBuildLinks : Deployment -> List String
+pullBuildLinks deployment =
+    case deployment.builds of
+        Nothing ->
+            []
+
+        Just builds ->
+            List.map (String.dropRight 1) (List.map (String.dropLeft 1) (List.map Debug.toString (List.map .link builds)))
+
+
+{-| linksView : takes list of links and creates an HTML msg that displays as a list of links
+-}
+linksView : List String -> Html msg
+linksView links =
+    links
+        |> List.map (\link -> a [ href link ] [ text (String.append "#" (getHead (List.head (List.reverse (String.split "/" link))))) ])
```

Comment:
Suggestion: rewrite the build number + "#" construction to use `|>` (pizza slices). i find it more readable.
```elm
link
  |> String.split "/"
  |> List.reverse
  |> List.head
  |> Maybe.withDefault ""
  |> String.append "#"
```
(using `Maybe.withDefault` we can also drop the `getHead` function)

Another thought is to leverage [Route.match](https://github.com/go-vela/ui/blob/c1c1e1701c18b8063162f2a956f44eb5624fb764/src/elm/Routes.elm#L102) to map the string to a build link and then extract the right part. Then we get some validation that the provided link matches a build link pattern.

Reply:
I went with Maybe.withDefault and was able to clean up the code quite a bit! I can look into Route.match if you think that would be better